### PR TITLE
Feedback fixes

### PR DESCRIPTION
**Pull request by ep09772**
Fixed a majority of suggestions given by Xiaokang. There were a few that I responded to him about otherwise.

### CODE REVIEW

File: dagmc_stats.py [hunk 1]
```diff
@@ -48,6 +47,204 @@ def get_entity_ranges(my_core, meshset, entity_types):
...
+    
+    outputs
+    -------
+    mean : the median of the dataset 
+    
+    """
+    
+    #finds the number of data points
+    length = 0
+    for value in sorted_dict.values():
```
Comment (zxkjack123) [inline dagmc_stats.py]: You use `sum(sorted_dict.values())` in line 175, why not here?

File: test_dagmc_stats.py [hunk 2]
```diff
@@ -7,17 +7,74 @@
...
+    
+def test_median():
+    my_mean = dagmc_stats.find_mean(test_set)
+    values_list = list()
+    for val, count in test_set.items():
+        for count in range(count):
+            values_list.append(val)
+    computed_mean = statistics.mean(values_list)
+    assert(my_mean == computed_mean)
+
```
Comment (zxkjack123) [inline test_dagmc_stats.py]: Too many blank lines here.

File: dagmc_stats.py [hunk 3]
```diff
@@ -48,6 +47,204 @@ def get_entity_ranges(my_core, meshset, entity_types):
...
+    triangles_per_vertex_stats['minimum'] = min(sorted_dict.keys())
+    triangles_per_vertex_stats['maximum'] = max(sorted_dict.keys())
+    median = find_median(sorted_dict)
+    triangles_per_vertex_stats['median'] = median
+    mean = find_mean(sorted_dict)
+    triangles_per_vertex_stats['average'] = mean
+    return triangles_per_vertex_stats
+
+
+
```
Comment (zxkjack123) [inline dagmc_stats.py]:
```suggestion
```
PEP8 suggests 2 blank lines between functions.

**Comment by zxkjack123** [comment]
You don't need to open a new PR when there are some things. Just add the changes according to feedback as another commit and push it to the current branch. Github will automatically update your PR. For example, you can add your changes to branch `feedback_fixes`, no need to make another branch such as `feedback_fixes_2`.

File: dagmc_stats.py (hunk 1 above)
Comment (ep09772) [reply dagmc_stats.py]: I actually realized that after I responded to your comment!

**Pull request closed without merging**